pyfr/integrators/dual/pseudo/pseudosteppers.py

```diff
@@ -90,7 +88,7 @@ def _pseudo_stepper_has_lerrest(self):
 
     @property
     def _stepper_nfevals(self):
```

Comment:
Should it still be called _stepper_nfevals?

Reply:
changed to _pseudo_stepper_nfevals.